test/protocols/test_unit_conversion_time_conflict.txt

```diff
@@ -0,0 +1,83 @@
+documentation {
```

Comment:
Can we get rid of the irrelevant comments from test protocols and focus on the bits that are actually key to the test?